webpack.config.js

```diff
@@ -44,7 +48,7 @@ module.exports = {
           {
             loader: 'file-loader',
             options: {
-              outputPath: 'assets/',
+              name: 'assets/[name].[ext]',
```

Comment:
This change was so the assets don't get renamed to a hash every time since we don't wipe the docs folder.